scripts/getGitCommit.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
Can we get a

```sh
set -euo pipefail
```

here?  I think we should include it in every new script by default.

Reply:
good catch, thx